cpp/src/aztec/stdlib/recursion/verification_key/verification_key.hpp

```diff
@@ -248,21 +232,7 @@ template <typename Curve> struct verification_key {
         std::vector<barretenberg::fr> key_witnesses;
         key_witnesses.push_back(compressed_domain);
         key_witnesses.push_back(key->num_public_inputs);
-        for (const auto& [tag, selector] : key->constraint_selectors) {
-            const auto x_limbs = split_bigfield_limbs(selector.x);
-            const auto y_limbs = split_bigfield_limbs(selector.y);
-
-            key_witnesses.push_back(x_limbs[0]);
```

Comment:
Where did this logic go?

Reply:
There were two identical loops for handling the permutation_selectors and constraint_selectors. This gets to the heart of why it made sense to reduce everything to a single map:`commitments`

Reply:
Oh sorry, the diff was displayed in a way that made me think the resulting loop didn't have the `key_witnesses.push_back`s.